### PR TITLE
[Styles] Contributors Page

### PR DESCRIPTION
**Pull request by runyasak**
# Screenshot

image

# Scope of work
Update styles to Contributors page by adding for space with padding and implement flex layout.

# Discussion
I have a plan to change the color of contributor username or `login` to be more readable.
I would like to use `props.theme.colors.primary` and it will be responsive with theme.
Actually, it would be error for typing.

Screen Shot 2564-04-05 at 01 03 58

It would be appreciated if you give me advices or suggestions. 😁

### CODE REVIEW

**Comment by aaronamm** [comment]
@runyasak Thanks for your contribution, I will check with that error by tonight.

File: src/components/MdxComponents/Contributors.tsx [hunk 1]
```diff
@@ -37,7 +47,12 @@ const Contributors = () => {
         return (
           // eslint-disable-next-line react/jsx-no-target-blank
           <a key={contributor.id} href={contributor.html_url} target='_blank'>
-            <img alt={contributor.login} src={contributor.avatar_url} />
+            <img
+              alt={contributor.login}
+              src={contributor.avatar_url}
+              width="100px"
```
Comment (aaronamm) [inline src/components/MdxComponents/Contributors.tsx]: Why do we need image's width, height attribute? I think we can set it from CSS component.
Could you please check it?

Comment (runyasak) [reply src/components/MdxComponents/Contributors.tsx]: It can make it both. But I will follow your guide by convert to CSS component. 😁

**Comment by aaronamm** [comment]
Here is the change I made and it works.
image


**Dark theme**
image

**Light theme**
image

**Comment by runyasak** [comment]
> Here is the change I made and it works.
> image
> 
> **Dark theme**
> image
> 
> **Light theme**
> image

@aaronamm 
Thank you, for suggestion. 😁 🙏

**Comment by aaronamm** [comment]
@runyasak After you've apply `props.theme.colors.primary`, I will merge your PR.
Thanks.

**Comment by runyasak** [comment]
I have resolved them.
Please, review my commits again. 😁
@aaronamm

**Comment by aaronamm** [comment]
Merged, thanks for your contribution.

**Pull request merged**